app/controllers/foreman_tasks/api/tasks_controller.rb

```diff
@@ -311,6 +323,13 @@ def tasks_list(filtered_scope)
           results: results
         }
       end
+
+      def bulk_scope
+        scope = resource_scope
+        scope = scope.search_for(params[:search]) if params[:search]
+        scope = scope.select('DISTINCT foreman_tasks_tasks.*')
+        scope.where(:id => params[:task_ids]) if params[:task_ids]
```

Comment:
If `params[:task_ids]` is `nil`, this method would return `nil`. Just add `scope` as the last line